[14.0][FIX]stock_orderpoint_generator: update views to modern format

PicchiSeba opened the pull request:
Fixes the following warnings during project build
- `DeprecationWarning: The <act_window> tag is deprecated, use a <record> for 'act_create_product_conf'. warnings.warn(f"The <act_window> tag is deprecated, use a <record> for {xml_id!r}.", DeprecationWarning)`
- `DeprecationWarning: The <act_window> tag is deprecated, use a <record> for 'act_create_product_template_conf'. warnings.warn(f"The <act_window> tag is deprecated, use a <record> for {xml_id!r}.", DeprecationWarning)`

rousseldenis commented:
@PicchiSeba Could you rebase ? Thanks

jbaudoux reviewed (commented):
Can you drop changes to OCA generated files ?

You can use 
`export SKIP=oca-gen-addon-readme`
to prevent those updates

PicchiSeba committed:
[FIX]stock_orderpoint_generator: update views to modern format

Fixes the following warnings during project build
- `DeprecationWarning: The <act_window> tag is deprecated, use a <record> for 'act_create_product_conf'.
  warnings.warn(f"The <act_window> tag is deprecated, use a <record> for {xml_id!r}.", DeprecationWarning)`
- `DeprecationWarning: The <act_window> tag is deprecated, use a <record> for 'act_create_product_template_conf'.
  warnings.warn(f"The <act_window> tag is deprecated, use a <record> for {xml_id!r}.", DeprecationWarning)`

PicchiSeba commented:
Done @jbaudoux

jbaudoux commented:
/ocabot merge patch
